test/e2e/lib/gutenberg/gutenberg-editor-sidebar-component.js

```diff
@@ -223,11 +223,13 @@ export default class GutenbergEditorSidebarComponent extends AsyncBaseContainer
 			this.driver,
 			By.css( '.edit-post-post-visibility__toggle' )
 		);
-		await driverHelper.selectElementByText(
+		await this.driver.sleep( 1000 ); // wait for popover to be fully loaded
+		await driverHelper.setCheckbox(
 			this.driver,
-			By.css( '.editor-post-visibility__dialog-label' ),
-			'Private'
+			By.css( 'input#editor-post-private-0[value="private"]' )
 		);
+
+		await driverHelper.waitForAlertPresent( this.driver );
```

Comment:
Is this alert always going to be there? I thought it just wasn't clicking the checkbox like it should

Reply:
nevermind... forgot to read description closely